readthedocs/domaindata/admin.py

```diff
@@ -0,0 +1,13 @@
+"""Domain Admin classes."""
+from django.contrib import admin
+
+from .models import DomainData
+
+
+class DomainDataAdmin(admin.ModelAdmin):
```

Comment:
`DomainData` name conflicts with our `Domain` model to me.

Considering that we are talking about Sphinx Domains here, I'd name all of these ones (including the Django App) as `SphinxDomain`. I think it will be better to get the meaning immediately when re reading about this.